packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
```

Comment:
How about "See the [Databricks documentation](https://docs.databricks.com/dev-tools/vscode-ext.html) for complete usage information."

(Or